.github/workflows/e2e_coder1_grpo.yml

```diff
@@ -0,0 +1,62 @@
+name: e2e_coder1_grpo
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the main branch
+  push:
+    branches:
+      - main
+      - v0.2.x
+    paths:
+      - "**/*.py"
+      - .github/workflows/e2e_coder1_grpo.yml
+  pull_request:
+    branches:
+      - main
+      - v0.2.x
+    paths:
+      - "**/*.py"
+      - .github/workflows/e2e_coder1_grpo.yml
+      - "tests/e2e/*.sh"
```

Comment:
could you limit the test trigger path to include the new script added by you, instead of including all scripts under `e2e`?

Reply:
please remove the push triggers and keep the pull_request triggers. Our CI is at capacity at the moment. 
Do you mind also temporarily removing all other yml files so that we can test your new yml for a few rounds.